src/app.js

```diff
@@ -1,16 +1,29 @@
-const express = require("express");
+const LogStream = require("./myStream");
 const bodyParser = require("body-parser");
+const errorHandler = require("./error-handler");
+const express = require("express");
+const morgan = require("morgan");
 const repositoryCotroller = require("./controllers/repository");
+const userCotroller = require("./controllers/user");
+const { HttpNotFound } = require("./utils/errors");
 
+const myStream = new LogStream();
 const app = express();
 
+app.use(morgan("dev", { stream: myStream }));
 app.use(bodyParser.json());
+
 app.get("/", (_req, res) => {
   res.json({ message: "Hello world" });
 });
+app.get("/repositories/:username", repositoryCotroller.getRepoStar);
```

Comment:
```suggestion
app.get("/repositories/:username", RepositoryCotroller.getRepoStar);
```

Para modulos eh legal importar eles em letra maiscula para diferenciar de variaveis normais.